src/rpc/misc.cpp

```diff
@@ -574,7 +574,7 @@ uint32_t getCategoryMask(UniValue cats) {
     for (unsigned int i = 0; i < cats.size(); ++i) {
         uint32_t flag = 0;
         std::string cat = cats[i].get_str();
-        if (!GetLogCategory(&flag, &cat)) {
+        if (!GetLogCategory(&flag, &cat) || (cat == "0" || cat == "1")) {
```

Comment:
According to the help text, this is a valid "category":

```
  -debug=<category>
       Output debugging information (default: 0, ...

Reply:
@MarcoFalke Thank you for your review.
Oh, indeed.
I think `-debug=1`(or0) is very clear for initial parameter, but I'm afraid that it is a little bit hard to understand that `"0"` is ignored in this RPC command.
So, I will add `{BCLog::NONE, "none"}` to `LogCategories[]` and add to the help text that `"0"` and `"1"` are the shorthand of `"none"` and `"all"`.

Reply:
I wasn't even aware of this rpc. You might want to wait for feedback by someone who used this.

Reply:
Then, I will wait for feedback for a while, thanks.